### ISSUE TITLE
[Bug]: Still having ffmpeg issues

### ISSUE THREAD

**Issue by JRiehl728** [elebumm/RedditVideoMakerBot#1774]
### Describe the bug

Having this error-

│ Sorry, something went wrong with this version! Try again, and feel free to report this issue at GitHub or the       │
│ Discord community.                                                                                                  │
│ Version: 3.2                                                                                                        │
│ Error: ffmpeg error (see stderr output for detail)                                                                  │
│ Config: {'allow_nsfw': False, 'theme': 'dark', 'times_to_run': 1, 'opacity': 0.9, 'storymode': False,               │
│ 'storymodemethod': 1, 'storymode_max_length': 1000, 'resolution_w': 1080, 'resolution_h': 1920, 'zoom': 1.0,        │
│ 'background': {'background_video': 'rocket-league', 'background_audio': 'lofi', 'background_audio_volume': 0.15,    │
│ 'enable_extra_audio': False, 'background_thumbnail': False, 'background_thumbnail_font_family': 'arial',            │
│ 'background_thumbnail_font_size': 96, 'background_thumbnail_font_color': '255,255,255'}, 'tts': {'voice_choice':    │
│ 'tiktok', 'random_voice': True, 'elevenlabs_voice_name': 'Bella', 'elevenlabs_api_key': 'REDACTED',                 │
│ 'aws_polly_voice': 'Matthew', 'streamlabs_polly_voice': 'Matthew', 'tiktok_voice': 'en_us_001', 'tiktok_sessionid': │
│ 'REDACTED', 'python_voice': '1', 'py_voice_num': '2', 'silence_duration': 0.3, 'no_emojis': False}}                 │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
Traceback (most recent call last):
  File "C:\Users\name\OneDrive\Documents\RedditTokAutomator\RedditVideoMakerBot-master\main.py", line 134, in <module>
    raise err
  File "C:\Users\name\OneDrive\Documents\RedditTokAutomator\RedditVideoMakerBot-master\main.py", line 114, in <module>
    run_many(config["settings"]["times_to_run"])
  File "C:\Users\name\OneDrive\Documents\RedditTokAutomator\RedditVideoMakerBot-master\main.py", line 69, in run_many
    main()
  File "C:\Users\name\OneDrive\Documents\RedditTokAutomator\RedditVideoMakerBot-master\main.py", line 61, in main
    make_final_video(number_of_comments, length, reddit_object, bg_config)
  File "C:\Users\name\OneDrive\Documents\RedditTokAutomator\RedditVideoMakerBot-master\video_creation\final_video.py", line 376, in make_final_video
    ffmpeg.output(
  File "C:\Users\name\AppData\Local\Programs\Python\Python310\lib\site-packages\ffmpeg\_run.py", line 325, in run
    raise Error('ffmpeg', out, err)
ffmpeg._run.Error: ffmpeg error (see stderr output for detail)
Progress:   0%|

-----------------------------------------------
Still having this ffmpeg issue people are seeing often.
So far ive tried the uninstall ffmpeg and ffmpeg-python and reinstall just ffmpeg-python. I've installed ffmpeg globally, and I've added the Video Maker folder as a system environment variable.

Not sure what else to try at this point.

Any input would be great :)

### Reproduction Steps

Normal Installation but so far ive tried the uninstall ffmpeg and ffmpeg-python and reinstall just ffmpeg-python. I've installed ffmpeg globally, and I've added the Video Maker folder as a system environment variable.

### Expected behavior

The video to output properly

### System Information

Operating System : Windows 10
Python version : Python 3.10.6
App version / Branch : latest

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.

**Comment by beckettech** [comment]
I have the same exact error, have not found any solutions.
![Screenshot 2023-08-04 101709](https://github.com/elebumm/RedditVideoMakerBot/assets/71792674/a945395d-b577-4d57-9317-760d68c89bb1)

**Comment by Cug-21** [comment]
Likewise, need update, i was unsure if it was something with ffmpeg being in the wrong interpreter path but re installed in the correct one with no change in the error.

**Comment by beckettech** [comment]
@JRiehl728 @Cug-21 

Fixed my build by reading this issue: https://github.com/elebumm/RedditVideoMakerBot/issues/1739

Simple fix:
Download this version of ffmpeg and copy the 3 files in the 'bin' folder to your 'RedditVideoMakerBot' folder and replace the old files.

https://github.com/GyanD/codexffmpeg/releases/download/2023-06-27-git-9b6d191a66/ffmpeg-2023-06-27-git-9b6d191a66-full_build.7z

**Comment by Cug-21** [comment]
@beckettech you are a legend bro

**Comment by thisisrick25** [comment]
Please tell me if everyone elses final video renders with background lofi and audio?

**Comment by xJWzZ** [comment]
@thisisrick25 Yes mine renders correctly

**Comment by KikiManjaro** [comment]
I had exactly the same problem and here's how I fixed it:

I had an old version of ffmpeg installed on my machine that was added to my PATH. After uninstalling the old version and relaunching the script, it offered to install ffmpeg on its own, but I still got the same error.

This time, the files were present in the root, so I was able to replace them with those supplied by @beckettech  https://github.com/elebumm/RedditVideoMakerBot/issues/1774#issuecomment-1665829314 and that fixed my problem, thank you!
